iceoryx_posh/test/moduletests/test_roudi_memory_provider.cpp

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//#define protected public
```

Comment:
If commented out it has to be removed. It is really dangerous to use since changing the access specifier may change the data layout leading to undefined bevahiour during data access.

Do not introduce those kind of defines in new (test) code.

Legacy tests where this is still done need to be reworked in another issue.